### PR TITLE
45 profile portrait responsive

### PR DESCRIPTION
**Pull request by iamalx**
#45 Made profile page responsive: landscape and portrait mode for ipad; and for ipad to cell phone. Ionic components are hard to manage and work with. Spend a lot of time fixing bugs because of ionic component's predefined CSS properties.

### CODE REVIEW

File: src/pages/profile/profile.html [hunk 1]
```diff
@@ -21,39 +21,46 @@
...
-      <ion-col col-5 align-self-center>
+    <ion-row class="section" justify-content-center>
+      <ion-col  class="user-info-colum" align-self-center>
         <ion-list>
           <ion-item>
-            <ion-label color="dark">Name</ion-label>
-            <ion-input placeholder="Peter Horton" text-center></ion-input>
-          </ion-item>
+           
+            <ion-label class= "label"color="dark">Name</ion-label>
```
Comment (khoadnguyen) [inline src/pages/profile/profile.html]: @awstin17 did some work with fontawesome icons, perhaps we could use that styling here as well?

File: src/pages/profile/profile.scss [hunk 2]
```diff
@@ -18,5 +18,69 @@ page-profile {
...
+    .buttons-div {
+        display: flex;
+        justify-content: center;
+    }
+    .buttons1 {
+        width: 100px;
+        margin: 10px 10px auto 10px;
+    }
+   
+    @media screen and (min-width: 768px) {
```
Comment (khoadnguyen) [inline src/pages/profile/profile.scss]: Your styling formatting and media query layout is great, can you discuss your methodology in class during CR?

**Review by khoadnguyen — COMMENTED** [review]
Nice job. 🌊

File: src/pages/profile/profile.scss (hunk 2 above)
Comment (iamalx) [reply src/pages/profile/profile.scss]: sure!

**Pull request merged**